components/error_page/error_message.test.tsx

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+/// <reference types="Jest" />
```

Comment:
This is the only way I could get the tests working, as I mentioned in the description.

Reply:
I don't know the correct fix but this seems to be not the proper solution.  Cypress has its own package.json (inside /e2e folder).  The mentioned conflicts will disappear when packages are not installed or removed in inside /e2e folder which is the case in our CI pipeline.  

I suggest to revert this change until we have permanent fix (if necessary).